### PR TITLE
Moved all frontend files to the src folder

### PR DESCRIPTION
**Pull request by KarolinaKopacz**
Moved all frontend files to the src folder

### CODE REVIEW

File: src/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev -p 3003",
```
Comment (dborowiecki) [inline src/package.json:6]: I think we should keep 3003 to be consistent

**Review by dborowiecki — CHANGES_REQUESTED** [review]
looks good, i left one comment

**Comment by lruzicki** [comment]
Some changes in .circleci/config.yml may be required after moving package.json.
[here](https://github.com/GovStackWorkingGroup/testing-webapp/blob/develop/.circleci/config.yml#L16) you can change it to run yarn in the directory where the package.json is available.

**Review by lruzicki — APPROVED** [review]
Looks good

next time you can squash repeated commits simply with `git rebase -i HEAD~31`

**Review by dborowiecki — APPROVED** [review]

**Pull request merged**